code/modules/research/designs/smelting_designs.dm

```diff
@@ -41,6 +41,14 @@
 	build_path = /obj/item/stack/sheet/plastitaniumglass
 	category = list("initial")
 
+/datum/design/plarglass_alloy
+	name = "Reinforced Plasma + Glass alloy"
```

Comment:
I think I'd go with either "Plasma + Reinforced Glass alloy" or "Plasma + Iron + Glass alloy". It's not the plasma that's reinforced.